src/pages/XeniumAnalyser.tsx

```diff
@@ -103,6 +108,7 @@ const XeniumAnalyser = () => {
       .max(new Date(), 'Please select a date and time on or before current time')
       .required('Time is a required field')
       .label('Time'),
+    equipmentId: Yup.number().required().label('Equipment').required('Equipment is a required field'),
```

Comment:
Why `equipmentId`  is optional in form data structure and a required field in validation?

Reply:
so we can init the form with empty equipmentId option

Reply:
Ok